elasticsearch_faker/_es_client.py

```diff
@@ -103,7 +103,7 @@ def create_index(self, index_name: str, mapping_filepath: str) -> int:
             return errno.EACCES
         except ConnectionError as e:
             logger.error(e)
-            return errno.ECOMM
+            return errno.EDOM
```

Comment:
Could you explain why you are trying to change the error code from ECOMM to EDOM?